### PR TITLE
Make snippet work with dart_analysis_server

### PR DESCRIPTION
**Pull request by 50ways2sayhard**

### CODE REVIEW

File: lsp-bridge.el [hunk 1]
```diff
@@ -547,8 +547,7 @@ Then LSP-Bridge will start by gdb, please send new issue with `*lsp-bridge*' buf
                (if (minibufferp) (window-buffer (minibuffer-selected-window))
                  (current-buffer))
 
-             (let ((snippet-fn (and (string= kind "Snippet")
```
Comment (manateelazycat) [inline lsp-bridge.el:550]: Why delete (string= kind "Snippet") ?

**Review by manateelazycat — CHANGES_REQUESTED** [review]

File: lsp-bridge.el (hunk 1 above)
Comment (manateelazycat) [reply lsp-bridge.el:550]: But if you delete this line, snippet-fn always is true, we can't do lsp-bridge--apply-text-edits if Kind is auto-import.

Comment (50ways2sayhard) [reply lsp-bridge.el:550]: I see. I'll push a commit to fix it.

Comment (50ways2sayhard) [reply lsp-bridge.el:550]: @manateelazycat done

**Pull request merged**